src/system/permissions-users-all.md

```diff
@@ -29,39 +33,47 @@ _Example Admin Users_
 
         The User Name should be easy to remember. It is not case-sensitive. For example, if your user name is `John`, you can also log in as `john`.
 
-    - Complete the following information:
+    - Complete the following user information:
 
         - **First Name**
         - **Last Name**
         - **Email address**
 
-        This email address must be different from the one that is associated with your original Admin account.
+        Each user account requires a unique email address.
 
-    - Assign a **Password** to the account.
+    - Enter a **Password** for the account.
 
         {:.bs-callout-info}
-        An Admin password must be seven or more characters long, and include both letters and numbers. For additional password options, see [Configuring Admin Security]({% link stores/security-admin.md %}).
+        An Admin password must be seven or more characters long and include both letters and numbers. For additional password options, see [Configuring Admin Security]({% link stores/security-admin.md %}).
 
-    - In the **Password Confirmation** box, repeat the password to make sure it was entered correctly.
+    - For **Password Confirmation**, re-enter the password to make sure it was entered correctly.
 
     - If your store has multiple languages, set **Interface Locale** to the language to be used for the Admin interface.
 
 1. Set **This Account is** to `Active`.
 
-1. Under _Current User Identity Verification_, enter **Your Password**.
+1. Click the calendar icon to set the **Expiration Date** for the user account.
```

Comment:
Is the new expiration available in both Open Source and Commerce? Just want to make sure we don't need conditioning here.

Reply:
was unable to find any restrictions in the ticket trails and this is community development.